spark-jobs/src/test/scala/za/co/absa/enceladus/standardization/StandardizationParquetSuite.scala

```diff
@@ -18,7 +18,7 @@ package za.co.absa.enceladus.standardization
 import java.util.UUID
 
 import org.apache.spark.sql.DataFrame
-import org.apache.spark.sql.types._
+import org.apache.spark.sql.types.{StructField, _}
```

Comment:
Is there a reason for StructField to be out?

Reply:
No, not really. My import optimization somehow didn't catch it. Fixed.